packages/eclipse_cpp.rb

```diff
@@ -0,0 +1,69 @@
+# Adapted from Arch Linux AUR eclipse-cpp PKGBUILD at:
+# https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=eclipse-cpp
+require 'package'
+
+class Eclipse_cpp < Package
```

Comment:
I see what you're trying to do here.  Why not create an `Eclipse.rb` which allows the user to select which Eclipse version they would like to install like we do with the php and jdk packages?  That leads to another question.  Can these packages be installed concurrently or do they create a conflict with each other?  In the latter case, would it make sense to add logic to prevent installing the others to avoid conflicts?